app/models/motion.rb

```diff
@@ -55,15 +55,19 @@ class Motion < ActiveRecord::Base
   scope :chronologically,          -> { order('created_at asc') }
   scope :with_outcomes,            -> { where('motions.outcome IS NOT NULL AND motions.outcome != ?', '') }
 
-  scope :closing_soon_not_published, -> {
+  # recency threshold is the minimum length of time for a new motion_closing_soon event to
+  # be published. So, if the proposal is extended by 2 hours, we don't want to publish another
+  # motion closing soon event, but if it's extended by 10 days, we do.
+  scope :closing_soon_not_published, ->(timeframe, recency_threshold = 2.days.ago) do
      voting
-    .joins("LEFT OUTER JOIN events e ON e.eventable_id = motions.id AND e.eventable_type = 'Motion'")
```

Comment:
This join was the culprit. We don't use it, and it can cause the same motion to be selected twice (because it has multiple events)